CHANGELOG.md

```diff
@@ -47,6 +49,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#731](https://github.com/nix-rust/nix/pull/731))
 - Marked `pty::ptsname` function as `unsafe`
   ([#744](https://github.com/nix-rust/nix/pull/744))
+- Ability to cast types added to libc_enum!
```

Comment:
This is an internal API change and as such should not be documented in the CHANGELOG.